Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/MSBuildTools.cs

```diff
@@ -17,11 +17,43 @@ namespace Microsoft.Build.Unity.ProjectGeneration
 {
     public class MSBuildToolsConfig
```

Comment:
>MSBuildToolsConfig [](start = 17, length = 18)

Is this class intended to be used as a Unity-serialized object and a json serialized object?  From my reading I've only seen how it should be a .json file, in which case I don't see a purpose for the [SerializeField] attributes.

Reply:
For Unity's JSON serialization, you can use these attributes to serialize fields.

---
In reply to: [364906027](https://github.com/microsoft/MSBuildForUnity/pull/96#discussion_r364906027) [](ancestors = 364906027)